package.json

```diff
@@ -15,12 +15,29 @@
     "dev": "run-p watch start:syncserver dev:demo",
     "pub": "lerna publish --yes",
     "start:syncserver": "cross-env DEBUG='WebsocketServer' yarn workspace @automerge/example-sync-server start",
+    "repocheck": "manypkg check",
```

Comment:
you can now run `yarn repocheck` in the root yourself to check whether there are any problem/drifts on dependency versions.